docs/variants/msi_z690/test-matrix.md

```diff
@@ -75,6 +73,9 @@ Certification Procedure for both tested platforms:
 | 6.   | [BIOS lock support][BLS]                          | BLS           | All                                  |
 | 7.   | [SMM BIOS write protection][SMM]                  | SMM           | All                                  |
 | 8.   | [Early boot DMA protection][EDP]                  | EDP           | All                                  |
+| 9.   | [BIOS Setup password][PSW]                        | PSW           | All                                  |
```

Comment:
The nomenclature should be changed in accordance with the `Product IDs` documentation.

Reply:
Changed